sphinx/base/make_base.py

```diff
@@ -280,9 +260,9 @@ def get_dependencies(cls) -> Tuple[Type["Command"], ...]:
 
     @classmethod
     def _run(cls) -> None:
-        assert is_azure_build(), "Only implemented for Azure Pipelines"
-        # get current version of package in the form of folder/URL name (e.g., "1-0-0")
-        current_version = version_string_to_url(get_package_version())
+        if not is_azure_build():
+            raise RuntimeError("only implemented for Azure Pipelines")
+
         # remove docs build currently deployed, except for the docs versions folder
```

Comment:
@j-ittner @rkdy 
Should this section be adjusted, too, in case of `current_version != Versions().latest_non_rc_version`?

Otherwise, wouldn't a pre-existing root folder of docs be deleted howevere also now not replaced later?

Reply:
Indeed! Well spotted, thanks. Fixing this now.